CoreMapView/src/au/gov/asd/tac/constellation/views/mapview/MapViewTileRenderer.java

```diff
@@ -985,13 +986,4 @@ public String toString() {
             return String.format("%s: %d", name, count);
         }
     }
-
-    public boolean isIgnoreMapInteractions() {
```

Comment:
@cygnus-x-1 wouldn't it be better to keep this method as this bit of code is duplicated twice within this class? It can be instead `private` though.

Reply:
Perhaps - there are other code blocks just like it all throughout `MapViewTileRenderer`, the whole thing needs a clean up but that feels like a task for another day.

Reply:
True @cygnus-x-1 but this would be one less we would have had to do.